corehq/apps/app_manager/tests/test_suite_case_tiles.py

```diff
@@ -114,6 +114,19 @@ def add_columns_for_one_one_two_case_details(_module):
 class SuiteCaseTilesTest(SimpleTestCase, SuiteMixin):
     file_path = ('data', 'suite')
 
+    # Keeps the number of columns in parity with what mobile allows
+    def test_case_tile_column_count(self):
+        for choice in CaseTileTemplates.choices:
+            template_name = choice[0]
+            template_grid = case_tile_template_config(template_name).grid
+
+            for field in template_grid.values():
+                absWidth = field.get('x') + field.get('width')
```

Comment:
I think this should be `absolute_width` unless there's a clear reason for `absWidth`

Reply:
Agreed, no reason, just accidental camel case.